test/disabled_checks_test.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+require "test_helper"
+
+class DisabledChecksTest < Minitest::Test
+  def setup
+    @tracer_one = TracerCheck.new
+    @tracer_two = TracerCheck.new
+    @tracer_one.stubs(:code_name).returns('TracerOneCheck')
+    @tracer_two.stubs(:code_name).returns('TracerTwoCheck')
+    @visitor = ThemeCheck::Visitor.new(ThemeCheck::Checks.new([@tracer_one, @tracer_two]))
+  end
+
+  def test_ignore_all_checks
+    template = parse_liquid(<<~END)
+      {% comment %}theme-check-disable{% endcomment %}
+      {% assign x = 'hello' %}
+      {% comment %}theme-check-enable{% endcomment %}
+      hello
+    END
+    @visitor.visit_template(template)
+
+    assert_equal([
```

Comment:
I think asserting assign it not visited instead would make it clearer and less fragile to future changes:

```
refute_includes(@tracer_one.calls, :on_assign)
```

Same for other ones bellow.